assignments.Rmd

```diff
@@ -125,31 +126,30 @@ The following definitely are reasons to react positively to feedback:
 ### Bonus points
 
 In addition to the peergrade score, one can get bonus points from
-Chat activity and by answering a time usage questionnaire in the course page in mycourses.
-Chat activity and the questionnaire will not count for maximum assignment
-score.
+Chat activity and by answering a time usage questionnaire on the course page in mycourses.
+Chat activity and the questionnaire are not included in the maximum assignment
+score and thus are not required to receive a full 100\% score.
 
 Other students' Chat activity will not affect one's resulting
-grade, i.e. there is no need to try to perform in the Chat or to take
-any part in it at all.
+grade, i.e. there is no need to try to perform in the Chat or to take any part in it at all.
 
-After each assignment, a voluntary anonymous questionnaire is opened on [the course page in mycourses](https://mycourses.aalto.fi/course/view.php?id=28239) for the peergrading feedback period - Monday to Wednesday 23:59.
+After each assignment, a voluntary anonymous questionnaire is opened on [the course page in mycourses](https://mycourses.aalto.fi/course/view.php?id=28239) for the peergrading feedback period (Monday to Wednesday 23:59).
```

Comment:
we could clarify "(Monday 8:00 to Wednesday 23:59)" to not be misunderstood as "(Monday 23:59 to Wednesday 23:59)"